composer.json

```diff
@@ -25,14 +25,14 @@
         "ext-ctype": "*",
         "doctrine/cache": "^1.12.1 || ^2.1.1",
         "doctrine/collections": "^1.5",
-        "doctrine/common": "^3.0.3",
+        "doctrine/common": "^3.0.3@dev",
         "doctrine/dbal": "^2.13.1 || ^3.2",
         "doctrine/deprecations": "^0.5.3",
         "doctrine/event-manager": "^1.1",
         "doctrine/inflector": "^1.4 || ^2.0",
         "doctrine/instantiator": "^1.3",
         "doctrine/lexer": "^1.2.3",
-        "doctrine/persistence": "^2.4",
+        "doctrine/persistence": "^2.4 || ^3@dev",
```

Comment:
If you are fine with this PR, I'd suggest that we release Common 3.3 and Persistence 3.0 before merging it.

Reply:
Done.